k8s/ietfweb/nginx.conf

```diff
@@ -0,0 +1,53 @@
+worker_processes auto;
+pid /var/lib/nginx/nginx.pid;
+error_log /dev/stdout;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+        worker_connections 768;
+        # multi_accept on;
+}
+
+http {
+
+        ##
+        # Basic Settings
+        ##
+
+        sendfile on;
+        tcp_nopush on;
+        types_hash_max_size 2048;
+        # server_tokens off;
+
+        # server_names_hash_bucket_size 64;
+        # server_name_in_redirect off;
+
+        include /etc/nginx/mime.types;
+        default_type application/octet-stream;
+
+        ##
+        # SSL Settings
+        ##
+
+        ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3; # Dropping SSLv3, ref: POODLE
+        ssl_prefer_server_ciphers on;
+
+        ##
+        # Logging Settings
+        ##
+
+        access_log /dev/stdout;
+
+        ##
+        # Gzip Settings
+        ##
+
+        gzip on;
+
+        ##
+        # Virtual Host Configs
+        ##
+
+        include /etc/nginx/conf.d/*.conf;
+        include /etc/nginx/sites-enabled/*;
+}
```

Comment:
One minor thing - `8 spaces for indentation here vs 4 spaces in `nginx-default.conf`.

Reply:
I think 8 spaces are from the default `nginx` config.

Reply:
Addressed with https://github.com/ietf-tools/www/pull/453/commits/2e05a4daee167cce2c124d78f63e02ebab3ad62a